src/Illuminate/Mail/Transport/ArrayTransport.php

```diff
@@ -55,4 +54,14 @@ public function flush()
     {
         return $this->messages = new Collection;
     }
+
+    /**
+     * Get the string representation of the message.
```

Comment:
This is a transport class and not a message, so I'm not sure what this Docblock means?

Reply:
I took a look at the symfony transport an the `__toString()` always returns the DSN representation:

Amazon-Mailer:
https://github.com/symfony/amazon-mailer/blob/f3c83427da82748c27a762a68c5bc71364e598cb/Transport/SesApiAsyncAwsTransport.php#L28-L39

Postmark-Mailer:
https://github.com/symfony/postmark-mailer/blob/bd270db549b314e60da8991485137d68dbbd68de/Transport/PostmarkApiTransport.php#L44-L47

Currently there is no DSN representation for array and the log driver. So maybe just return the name would be enough here.

Maybe changing the docblock to the following would be better.

```suggestion
     * Get the string representation of the transport driver.
```